build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    kotlin("jvm") version "1.4.0"
+    id("org.jmailen.kotlinter") version "3.0.2"
```

Comment:
Will fail the build if your kotlin isn't formatted right, and you can use this to auto-fix it:

```
./gradlew formatKotlin
```